model/countries/DE/DE-parsing-rules.yaml

```diff
@@ -33,6 +33,19 @@ capture_definitions:
           prefix: {regex_reference: kHouseNumberOptionalPrefixRe}
           parts: [ {regex_reference: kBuildingValueRe} ]
 
+  AddressLinesDecomposition:
+    capture:
+      output: street-address-alternative-1
+      parts:
+      - capture:
+          output: street-location
+          parts: [ {regex_fragment: '.*'} ]
+      - regex_fragment: '\n'
```

Comment:
This currently assumes that a street-address-alternative-1 MUST have two lines.

Reply:
I guess that was the intent. If the street address has only one line, I think we should favor parsing with `StreetAddressDecomposition` via the decomposition_cascade.

Reply:
I see... can you leave a comment?

Should we add a third fallback in the cascade in case the street-address
1) consists only of a single line (AddressLinesDecomposition does not match) and 2) ParseStreetLocation does not match because of a lack of a house number?

Reply:
We could but I'm thinking whether it would add value. Are you thinking to copy the street address content into the street-name?

I will merge the PR for now to fix things before the raters run. We can amend the parsing expressions later depending on what we decide here.